### ISSUE TITLE
Add Text

### ISSUE THREAD

**Issue by Aphexus** [gburca/VirtMus#3]
It would be nice if one can add text(best I can see is one can do lines and rectangles). Such as using chord symbols, notes, etc.

It would need to allow for font sizes and possibly super and sub scripts(e.g., A^7b9_m would be an Am7 with a b9, the ^ raises to a super script and the _ lowers down a level).

**Comment by gburca** [comment]
The intention was to delegate anything more complicated than simple lines or rectangles to a dedicated application (ex: Inkscape or some other SVG editor) that has full support for arbitrary annotations. Inkscape is open source, cross platform and supported by VirtMus.

**Comment by Aphexus** [comment]
Yes, but simple text should be doable and is pretty useful to have.

Being able to quickly add a text note or chord symbol rather than opening an external editor saves  quite a bit of time unless no reloading has to be done(I'd imagine opening in an external editor requires reloading files).

Surely it shouldn't be much to also add a text graphic that can go along with the lines and rectangles?  Not too much extra code.
